### DISCUSSION TITLE
Json

### DISCUSSION THREAD

**Discussion by kachna96** [RicoSuter/NSwag#4092] [Q&A]
Hi, we are generating a CSharp client from openapi specification in .csproj like so

```
<ItemGroup>
    <OpenApiReference Include="V1\OpenApi\swagger.json">
      <CodeGenerator>NSwagCSharp</CodeGenerator>
      <Namespace>Xyz.Generated</Namespace>
      <ClassName>XyzClient</ClassName>
      <Options>/UseBaseUrl:false /GenerateClientInterfaces:true /JsonLibrary:SystemTextJson</Options>
    </OpenApiReference>
  </ItemGroup>
```

one issue with this is that JsonLibrary option doesn't do anything and it is still using Newtnsoft, can anyone advise if we are using it wrong or what's the correct syntax? The option should be [there](https://github.com/RicoSuter/NSwag/blob/209ad02e4123eeac7a092a61f6ad33f9b4cf0e84/src/NSwag.Commands/Commands/CodeGeneration/OpenApiToCSharpCommandBase.cs#L241)
Thanks!

**Comment by nZeus** [comment]
Have you tried to delete the file from the `obj` folder and rebuild the project?
This option works fine in our projects

**Comment by POnakS** [comment]
You need to alter `V1\OpenApi\swagger.json`, there is some kind of cache.
